src/stlink-lib/chipid.c

```diff
@@ -1061,7 +1060,7 @@ void dump_chips (void)
   struct stlink_chipid_params *ts;
   char *p, buf[100];
   FILE *fp;
-  
+
```

Comment:
```suggestion
  
```

Reply:
I'd prefer to avoid white space where not needed for improved readability.